internal/db/mailbox.go

```diff
@@ -70,27 +70,41 @@ func RenameMailboxWithRemoteID(ctx context.Context, tx *ent.Tx, mboxID imap.Labe
 	return nil
 }
 
-func DeleteMailboxWithRemoteID(ctx context.Context, tx *ent.Tx, mboxID imap.LabelID) error {
-	// get uid validity
+func DeleteMailboxWithRemoteID(ctx context.Context, tx *ent.Tx, mboxID imap.LabelID) (imap.UID, bool, error) {
```

Comment:
Can you document what the bool indicates?

Reply:
done